scene/debugger/scene_debugger.cpp

```diff
@@ -269,7 +282,8 @@ Error SceneDebugger::parse_message(void *p_user, const String &p_msg, const Arra
 
 	} else if (p_msg.begins_with("runtime_node_select_")) { /// Runtime Node Selection
 		if (p_msg == "runtime_node_select_setup") {
-			runtime_node_select->_setup();
+			ERR_FAIL_COND_V(p_args.is_empty() || p_args[0].get_type() != Variant::DICTIONARY, ERR_INVALID_DATA);
+			runtime_node_select->_setup(p_args[0].operator Dictionary());
```

Comment:
```suggestion
			runtime_node_select->_setup(p_args[0]);
```
You don't need to explicitly cast variants when assigning or passing.